### ISSUE TITLE
Connection Errors

### ISSUE THREAD

**Issue by nateabowman** [AlexSwift/SantosRP#4]
[ERROR] gamemodes/santosrp/gamemode/server/mysql/mysql_setup.lua:2: attempt to call field 'New' (a nil value)
Everything seems to be setup correctly.

**Comment by harry699** [comment]
These Are Not SantosRP Files Carry On Looking (:

**Comment by AlexSwift** [comment]
You're missing the correct mysql module. iirc correctly I made use of a tmysql3 wrapper to implement OO, but didn't release it.

@harry699 These are indeed the SantosRP files, however only from when I worked on them. These are actually the files from the very first release, so a lot has changed since when I worked with JAC Systems LTD. Sorry for the snidiness but I find your comment amusing as if you as some sort of authority. In reality you are just misinformed at best.

@chadreed93 if you are interested in hosting this revision for nostaligic reasons, I would be more than happy to continue development in a private branch for you to get you started - with a decent framework to start from at the very least. Reply to this thread if something like this is of interest to you.

**Comment by nateabowman** [comment]
Yeah that would be awesome if you wanted to. Does the current revision not work? Or how could I at least get it to connect with this revision? I am using libmysql.dll, gmsv_mysqloo v9, gmsv_tmysql4(Are both needed?), and the mysql wrapper as shown in the description. 
Thanks for the quick reply @AlexSwift

**Comment by AlexSwift** [comment]
@chadreed93 Only one should be needed. Where have you put the wrapper?

**Comment by nateabowman** [comment]
garrysmod\lua\includes\modules\mysql.lua was done first and is still there, and I also tried putting it in SantosRP mysql folder since it includes those at init anyway.

**Comment by AlexSwift** [comment]
Will test it out in the morning. Any console output would be helpful. I'd expect an error before that to get that error

**Comment by nateabowman** [comment]
I restarted the server fully and collected this log: https://pastebin.com/pyNd9vDW 
Thanks again

**Comment by AlexSwift** [comment]
Could you run the command "lua_run require([[mysql]])" in your srcds/server console?
Also, it might be easier to add you on discord so we can chat more interactively to solve this

**Comment by nateabowman** [comment]
Prints: require([[mysql]])...
ProFragHosting.Com#2179

**Comment by XtronHD** [comment]
Can i get your discord?

**Comment by HamDestroyer** [comment]
I am having the same problem

**Comment by AlexSwift** [comment]
https://discord.gg/azSANNT
